### PR TITLE
lxi-tools: fix glib error

### LINKED ISSUE #72: macOS install
I've managed to get LXI-Tools to install on macOS Ventura 13.5 (GUI disabled), from source and with a brew formula, by simply adding lua as a dependency.

I think enabling the GUI will come together also, however I currently getting this error when the install runs `postinstall.py`
```shell
❯ HOMEBREW_NO_INSTALL_FROM_API=1 brew install lxi-tools

==> Fetching lxi-tools
==> Downloading https://github.com/lxi-tools/lxi-tools/archive/refs/tags/v2.5.tar.gz
Already downloaded: /Users/rob/Library/Caches/Homebrew/downloads/7b774ec1dbd550cb2353654c4709d7c413439ebabdb1f138c134593ff35b2890--lxi-tools-2.5.tar.gz
==> meson setup build --prefix=/opt/homebrew/Cellar/lxi-tools/2.5 --libdir=/opt/homebrew/Cellar/lxi-tools/2.5/lib --buildtype=release --wrap-mode=nofallback
==> meson compile -C build --verbose
==> meson install -C build
Last 15 lines from /Users/rob/Library/Logs/Homebrew/lxi-tools/03.meson:
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'gtk-update-icon-cache'
Installing src/lxi to /opt/homebrew/Cellar/lxi-tools/2.5/bin
Installing src/lxi-gui to /opt/homebrew/Cellar/lxi-tools/2.5/bin
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/build/man/lxi.1 to /opt/homebrew/Cellar/lxi-tools/2.5/share/man/man1
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/data/io.github.lxi-tools.lxi-gui.desktop to /opt/homebrew/Cellar/lxi-tools/2.5/share/applications
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/data/io.github.lxi-tools.lxi-gui.appdata.xml to /opt/homebrew/Cellar/lxi-tools/2.5/share/metainfo
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/data/io.github.lxi-tools.lxi-gui.gschema.xml to /opt/homebrew/Cellar/lxi-tools/2.5/share/glib-2.0/schemas
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/data/icons/hicolor/scalable/apps/io.github.lxi-tools.lxi-gui.svg to /opt/homebrew/Cellar/lxi-tools/2.5/share/icons/hicolor/scalable/apps
Installing /private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/data/icons/hicolor/symbolic/apps/io.github.lxi-tools.lxi-gui-symbolic.svg to /opt/homebrew/Cellar/lxi-tools/2.5/share/icons/hicolor/symbolic/apps
Running custom install script '/private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/build-aux/meson/postinstall.py'
FAILED: install script '/private/tmp/lxi-tools-20230723-22992-165tvli/lxi-tools-2.5/build-aux/meson/postinstall.py' exit code 1, stopped
```

I suspect the script is just looking in the wrong place (or rather brew has put `gtk-update-icon-cache` in a different place).

An explanation of what this script does and requires would help and be much appreciated.

### PR DESCRIPTION
**Pull request by ghost**
This formula revision fixes an error introduced in the original formula: GLib-GIO-ERROR **: No GSettings schemas are installed on the system. We mistakenly removed the schemas to workaround an error in the brew CI. This revision regenerates the schema post install. This is simply a formula revision, not a patch release.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This formula revision fixes [an error introduced](https://github.com/lxi-tools/lxi-tools/issues/72#issuecomment-1672683207) in the original formula:
GLib-GIO-ERROR **: No GSettings schemas are installed on the system.
We mistakenly removed the schemas to workaround an error in the brew CI.
This revision regenerates the schema post install.
This is simply a formula revision, not a patch release.

### CODE REVIEW

File: Formula/lxi-tools.rb [hunk 1]
```diff
@@ -4,23 +4,15 @@ class LxiTools < Formula
...
-
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "0ad1eacf1e8ae219d14d1d0584f97da77fb4f866dcb2deb63b5a64278c21c492"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "74f67748c50df2d75290b85eee36792ea4c8eb1b767a1fe81802a0aacf1d8170"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "df3254a41c62da6bfa63e8d7da78e9fb45cc2347710a3fd09dafa91e5d027a62"
-    sha256 cellar: :any_skip_relocation, ventura:        "910d48ed42f6cdf76d6166df6343309d2e50a60550a44b7137cdf9d5000916be"
-    sha256 cellar: :any_skip_relocation, monterey:       "4349534c758dad9cf5cbae6f2d799d5fb3be25887cd69d683ce921890ab55be8"
-    sha256 cellar: :any_skip_relocation, big_sur:        "0f31f8dba717b9434e3798f090f68503aa633b44a4247bdc8a13a39c24961fcc"
-    sha256                               x86_64_linux:   "4f84ffcf6baa6e46f5fa3d3555496db4f9dd3f0815a7f4faee51dae80bd77ad2"
-  end
```
Comment (chenrui333) [inline Formula/lxi-tools.rb:16]: no need to touch bottle block

File: Formula/lxi-tools.rb [hunk 2]
```diff
@@ -4,23 +4,15 @@ class LxiTools < Formula
   url "https://github.com/lxi-tools/lxi-tools/archive/refs/tags/v2.6.tar.gz"
   sha256 "a36699387b40080ea9eb8b1abc14d843f5e7a33b3a62fcfedaea9cc54214bdc8"
   license "BSD-3-Clause"
-
-  bottle do
```
Comment (carlocab) [inline Formula/lxi-tools.rb:8]: Please don't remove the bottle block. Also, use the commit message
```
lxi-tools: fix glib error
```

**Review by carlocab — CHANGES_REQUESTED** [review]

**Comment by robcarruthers** [comment]
I'm not sure I have done everythiong correctly above, Please tell me if I need to do anything to get this to progress

**Comment by robcarruthers** [comment]
If I reinstate the bottle block as requested, I get the following when testing locally.
```
==> Fetching lxi-tools
==> Downloading https://ghcr.io/v2/homebrew/core/lxi-tools/manifests/2.6_1
#=#=-  #       #                                                                                                                                                                                                                                               curl: (22) The requested URL returned error: 404

Error: lxi-tools: Failed to download resource "lxi-tools_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/lxi-tools/manifests/2.6_1
```

**Comment by carlocab** [comment]
You need to pass `--build-from-source` as indicated in the contributing guide and the PR template to avoid that error.

**Comment by robcarruthers** [comment]
Could you please help me fix the failing check?

I do not understand how to fix the commit style or how to rebase to origin/master

**Comment by robcarruthers** [comment]
I'll open a new PR for the new release 2.7 and incorporate these changes.

**Pull request closed without merging**